documentation/modules/proc-installing-using-bundle.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-openshift.adoc
+
+[id='installing-using-bundle-{context}']
+= Installing {ProductName} using a YAML bundle
+
+The simplest way to install {ProductName} is to use the predefined YAML bundles. 
+
+.Prerequisites
+
+* (Optional) Replace namespace if you want to deploy to something other than `{ProductNamespace}` (and substitute `{ProductNamespace}` in subsequent steps):
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+sed -i 's/{ProductNamespace}/_<my-namespace>_/' install/bundles/enmasse-with-standard-authservice/*.yaml
+----
+
+.Procedure 
+
+ifeval::["{cmdcli}" == "oc"]
+. Login as as a user with cluster-admin 
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} login -u system:admin
+----
+
+. Create the project where you want to deploy {ProductName}
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} new-project {ProductNamespace}
+----
+endif::[]
+ifeval::["{cmdcli}" == "kubectl"]
+. Create the namespace where you want to deploy {ProductName}
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create namespace {ProductNamespace}
+{cmdcli} config set-context $(kubectl config current-context) --namespace={ProductNamespace}
+----
+
+. Create a certificate to use with the `standard` authentication service. For testing purposes, you can create a self-signed certificate:
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+mkdir -p standard-authservice-cert
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=standard-authservice.{ProductNamespace}.svc.cluster.local" -out standard-authservice-cert/tls.crt -keyout standard-authservice-cert/tls.key
+----
+
+. Create a secret with the `standard` authentication service certificate:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create secret tls standard-authservice-cert --cert=standard-authservice-cert/tls.crt --key=standard-authservice-cert/tls.key
+----
+
+. Create a certificate to use with the API server. For testing purposes, you can create a self-signed certificate:
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+mkdir -p api-server-cert/
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=api-server.{ProductNamespace}.svc.cluster.local" -out api-server-cert/tls.crt -keyout api-server-cert/tls.key
+----
+
```

Comment:
Add:
"where <{ProductNamespace}> is the namespace where {ProductName} is deployed."